main.py

```diff
@@ -0,0 +1,62 @@
+from fastapi import FastAPI, Depends
+from fastapi_pagination import add_pagination, Page, paginate
+from sqlalchemy.orm import Session
+from crud import get_many, get_one, update_item, get_books, add_book, add_item, delete_item
+from database import get_session
+from models import DBAuthor, DBBook
+from schemas import AuthorWithId, AuthorNoId, BookWithId, BookNoId
+
+app = FastAPI(title="Library Manager")
+
+
+@app.get("/authors/", response_model=Page[AuthorWithId])
+def read_authors(db_session: Session = Depends(get_session)):
```

Comment:
your endpoints don't use `session` at all. so maybe they should not even know about session?